src/map/lua/lua_baseentity.cpp

```diff
@@ -8734,6 +8734,17 @@ inline int32 CLuaBaseEntity::isSpellAoE(lua_State* L)
     return 1;
 }
 
+inline int32 CLuaBaseEntity::getGender(lua_State* L)
+{
+    DSP_DEBUG_BREAK_IF(m_PBaseEntity == nullptr);
+    DSP_DEBUG_BREAK_IF(m_PBaseEntity->objtype == TYPE_NPC);
```

Comment:
since you're casting it to a CharEntity, you should be checking if it's not a TYPE_PC